packages/@lwc/integration-types/src/playground/main.ts

```diff
@@ -0,0 +1,5 @@
+import { createElement } from 'lwc';
+import App from 'x/app';
+
+const elm = createElement('x-app', { is: App });
+document.body.appendChild(elm);
```

Comment:
I would love for us to validate here that we can actually call an `@api` method on `App` and access an `@api` property on it. This is a really common scenario, especially in unit tests.

Reply:
Now tested in `packages/@lwc/integration-types/src/create-element.ts`